src/apps/intel_mp/test_10g_2q_blast_vmdq_auto.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# Test VMDq with automatic pool selection
+
+SNABB_SEND_BLAST=true ./testsend.snabb $SNABB_PCI_INTEL1 0 source2.pcap &
+BLAST=$!
+
+SNABB_RECV_SPINUP=2 SNABB_RECV_DURATION=5 ./testvmdqrecv.snabb $SNABB_PCI_INTEL0 "90:72:82:78:c9:7a" nil nil 0 > results.0 &
+PID1=$!
+SNABB_RECV_SPINUP=2 SNABB_RECV_DURATION=5 ./testvmdqrecv.snabb $SNABB_PCI_INTEL0 "12:34:56:78:9a:bc" nil nil 1 > results.1
+
+wait $PID1
+kill -9 $BLAST
+
+[[ `cat results.* | grep "^RXDGPC" | awk '{print $2}'` -gt 10000 ]] &&\
+# both queues should see packets
```

Comment:
Again, I don't think you can insert comments inside lines broken by escaped newlines like this.